source/gui/settingsDialogs.py

```diff
@@ -1913,6 +1913,16 @@ def makeSettings(self, settingsSizer):
 		self.trapNonCommandGesturesCheckBox = sHelper.addItem(wx.CheckBox(self, label=trapNonCommandGesturesText))
 		self.trapNonCommandGesturesCheckBox.SetValue(config.conf["virtualBuffers"]["trapNonCommandGestures"])
 
+		# Translators: This is the label for a checkbox in the
+		# browse mode settings panel.
+		autoFocusFocusableElementsText = _("Automatically set system &focus to focusable elements")
+		self.autoFocusFocusableElementsCheckBox = sHelper.addItem(
+			wx.CheckBox(self, wx.ID_ANY, label=autoFocusFocusableElementsText)

```

Comment:
The use of `wx.ID_ANY` is not required.
```suggestion
			wx.CheckBox(self, label=autoFocusFocusableElementsText)
```